### PR TITLE
ZEK-1661-Add-Line-and-Section-Title-to-form-component

### PR DESCRIPTION
**Pull request by Yusuf-00**
added line and label inputs

### CODE REVIEW

File: src/components/form/Form.vue [hunk 1]
```diff
@@ -161,6 +163,24 @@ export default {
...
+                                data: input,
+                            }
+                        })
+                    }
+                    if(input.type == 'line') {
+                        columns.push({
+                            columnWidth: input.columnWidth || 12,
+                            content: {
+                                component: 'html',
+                                data: input,
```
Comment (AbdAsh) [inline src/components/form/Form.vue]: @Yusuf-00 if there is no input for the data please give it default content for the sake of ease of usage.

File: src/stories/Form.stories.js [hunk 2]
```diff
@@ -136,9 +151,11 @@ Primary.args = {
                 textPosition: "left"
             },
         ]
-        
+
     }
   ],
+  sectionLabel: "balbla",
+  formSection: "lol",
```
Comment (AbdAsh) [inline src/stories/Form.stories.js]: Do these have any usage?

**Review by AbdAsh — CHANGES_REQUESTED** [review]

File: src/stories/Form.stories.js (hunk 2 above)
Comment (Yusuf-00) [reply src/stories/Form.stories.js]: no was for testing forgot to remove them

**Review by AbdAsh — CHANGES_REQUESTED** [review]
The types `line` and `title` are returning on submission, we should avoid that as They have no actual values

File: src/components/form/Form.vue [hunk 3]
```diff
@@ -243,6 +267,8 @@ export default {
     },
     methods: {
         submitForm() {
+
+            console.log(this.formData)
```
Comment (AbdAsh) [inline src/components/form/Form.vue]: Remove log

File: src/components/form/Form.vue [hunk 4]
```diff
@@ -161,6 +167,24 @@ export default {
...
+                                data: input || {type: 'label', text: "label"},
+                            }
+                        })
+                    }
+                    if(input.type == 'line') {
+                        columns.push({
+                            columnWidth: input.columnWidth || 12,
+                            content: {
+                                component: 'html',
+                                data: input ||  { type: 'line', content: `<hr>`, styleObj: { color: "#000000"}},
```
Comment (AbdAsh) [inline src/components/form/Form.vue]: Why are you adding the `type` here again inside the data of the component

File: src/components/form/Form.vue [hunk 5]
```diff
@@ -161,6 +167,24 @@ export default {
                     if(input.type == 'captcha') {
                         this.containsCaptcha = true;
                     }
+                    if(input.type == 'label') {
+                        columns.push({
+                            columnWidth: input.columnWidth || 12,
+                            content: {
+                                component: 'text',
+                                data: input || {type: 'label', text: "label"},
```
Comment (AbdAsh) [inline src/components/form/Form.vue]: Why are you adding the `type` here again inside the data of the component

**Review by AbdAsh — CHANGES_REQUESTED** [review]

File: src/components/form/Form.vue (hunk 5 above)
Comment (Yusuf-00) [reply src/components/form/Form.vue]: my bad , I didn't realize that inputTypes already add them , will remove them @AbdAsh

**Review by AbdAsh — APPROVED** [review]
LGTM

**Pull request merged**